assets/schema_input.json

```diff
@@ -46,7 +46,14 @@
                 "type": "string",
                 "pattern": "^\\S+$",
                 "unique": ["patient", "sample"],
-                "dependentRequired": ["fastq_1"],
+                "anyOf": [
```

Comment:
How is one suppose to understand that?  If `lane` is present, then `fastq_1` or `bam` should also be present?